integration-tests/src/test/resources/oke/terraform/kube_config.tf

```diff
@@ -3,15 +3,21 @@
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 */
 
-variable "cluster_kube_config_expiration" { default = 2592000 }
-variable "cluster_kube_config_token_version" { default = "1.0.0" }
+variable "cluster_kube_config_expiration" {
+  default = 2592000
+}
+
+variable "cluster_kube_config_token_version" {
+  default = "1.0.0"
```

Comment:
Is the 1.0 token still supported. They are on 2.0, correct?

Reply:
switched to 2.0